src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -17,107 +32,215 @@ type PipelineRunDetailsTabProps = {
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Message</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun.status?.conditions[0]?.message ?? '-'}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Log snippet</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <CodeBlock>
+                    <CodeBlockCode id="code-content">
+                      {pipelineRun.status?.taskRuns[`${pipelineRun.metadata?.name}-show-summary`]
+                        ?.status?.taskSpec?.steps[0].script ?? '-'}
+                    </CodeBlockCode>
+                  </CodeBlock>
+                  <Button
+                    variant="link"
+                    isInline
+                    component={(props) => (
+                      <Link
+                        {...props}
+                        to={`/app-studio/pipelineruns/${pipelineRun.metadata?.name}?activeTab=logs`}
+                      />
+                    )}
+                  >
+                    See logs
+                  </Button>
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Pipeline</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun.metadata?.labels[PipelineRunLabel.PIPELINE_NAME] ?? '-'}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Application</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Link
+                    to={
+                      `/app-studio/applications/${
+                        pipelineRun.metadata?.labels[PipelineRunLabel.APPLICATION]
+                      }` ?? ''
+                    }
+                  >
+                    {pipelineRun.metadata?.labels[PipelineRunLabel.APPLICATION] ?? '-'}
+                  </Link>
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Source</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <ExternalLink
+                    href={
+                      pipelineRun.metadata?.annotations[
+                        PipelineRunLabel.COMMIT_FULL_REPO_URL_LABEL
+                      ] ?? ''
+                    }
+                  >
+                    {pipelineRun.metadata?.annotations[
+                      PipelineRunLabel.COMMIT_FULL_REPO_URL_LABEL
+                    ] ?? '-'}
+                  </ExternalLink>
```

Comment:
You'll want to omit the link altogether if there's no annotation.

Reply:
done, ty